src/DevTool.js

```diff
@@ -9,7 +9,9 @@ export default class DevTool extends Component {
     static propTypes = {
         highlightTimeout: PropTypes.number,
         position: PropTypes.object,
-        noPanel: PropTypes.bool
+        noPanel: PropTypes.bool,
```

Comment:
Would you mind setting `defaultProps` for these properties? Without, afaik, you will see for example `undefined` as `class` property

Reply:
Good catch! How about this way? 
`<div clasName={className || ''} />`

Reply:
that would also work, but `defaultProps` are kinda invented for this :)

Reply:
👌